R/dfm_weight.R

```diff
@@ -230,6 +230,7 @@ dfm_smooth.default <- function(x, smoothing = 1) {
 dfm_smooth.dfm <- function(x, smoothing = 1) {
     x <- as.dfm(x)
     if (!nfeat(x) || !ndoc(x)) return(x)
+    x@smooth <- x@smooth + smoothing
```

Comment:
This is a policy decision, but I think this is the natural way for this to occur: we cumulate subsequent smoothing operations, so that the `@smooth` slot records the total smoothing applied to the _original_ dfm. @koheiw do you agree?